### PR TITLE
[WIP] M3O post

### PR DESCRIPTION
**Pull request by crufter**

### CODE REVIEW

File: blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Micro is and always was, an opinionated framework and ecosystem. Convention over configuration. Easy bootstrapping with zero dependencies locally. Filling in blanks as demands of scaling and resiliency comes up - by switching out implementations of interfaces with more sophisticated ones - that was always the Micro way.
+
+With M3O we plan to keep this approach, working both ways: moving things from local to prod, or prod to local should be a breeze. Micro is particularly well suited for this as an ecosystem built around the use of pluggable abstractions. One of our main goals is to make handling multi environments as easy as possible - regardless of where and how the services are being run, managing and interacting with them should feel just as native as running them on your laptop or PC.
+
+We try to make it so when someone learns how to use Micro locally, deploying to M3O will not be further away then a CLI command (assuming an account on M3O exists). This reuse of the well known and already useful things that are part of the daily workflow will hopefully provide the easiest way for developers to migrate from local to prod or vice versa.
```
Comment (crufter) [inline blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md:59]: I think we can remove `(assuming an account on M3O exists).` breaks the flow

File: blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Somehow between the disparate contribution of technologies to the Cloud ecosystem from startups and the thousands of features now promoted by AWS, the ease of use for the developer has drastically suffered. In fact we argue, the cognitive load on the developer is now so bad that we cannot even begin to start building software for the cloud without 3-6 months of wrangling a kubernetes cluster on one of the incumbent cloud providers, while muddling our way through CloudFormation and walls of YAML...
+
+Or perhaps it's a form of decision paralysis due to the explosion of available technologies. In this fast moving industry one often can't help but follow the technologies provided by FAANGs in a "Nobody got fired for using IMB" spirit. Or the thousand open source projects listed on the CNCF landscape which give us FOMO and a headache all a the same time as we figure out if any of its needed or all of it.
```
Comment (domwong) [inline blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md:22]:
```suggestion
Or perhaps it's a form of decision paralysis due to the explosion of available technologies. In this fast moving industry one often can't help but follow the technologies provided by FAANGs in a "Nobody got fired for using IBM" spirit. Or the thousand open source projects listed on the CNCF landscape which give us FOMO and a headache all a the same time as we figure out if any of its needed or all of it.
```

File: blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Naturally, with millions of us working in this industry, we can't and don't expect everything to go the way we envision - we just enthusiastically wish for it. Our industry is still very young, and we all work hard to make it and the world a better place. That being said, we hope our hard work and decades of collective experience with microservices will result in something that the users will love and put to productive use.
+
+## Who are we?
+
+We are a collective of engineers who's experienced the woes of cloud-native complexity, built platforms before the era of containers and fire fought battles with Kubernetes on multiple cloud providers. We are the every-man, the every-day engineer, who just so happens to now want to do something to combat the complexity that AWS, Google and others have introduced to the world.
```
Comment (domwong) [inline blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md:53]:
```suggestion
We are a collective of engineers who have experienced the woes of cloud-native complexity, built platforms before the era of containers and fire fought battles with Kubernetes on multiple cloud providers. We are the every-man, the every-day engineer, who just so happens to now want to do something to combat the complexity that AWS, Google and others have introduced to the world.
```

File: blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md (hunk 2 above)
Comment (crufter) [reply blog/web/content/_posts/2020-05-05-m3o-micro-services-development-platform.md:22]: haha! nice catch

**Pull request merged**